src/promptflow-evals/pyproject.toml

```diff
@@ -45,8 +45,13 @@ aiohttp_retry = ">=2.8.3"
 websocket-client = ">=1.2.0"
 jsonpath_ng = ">=1.5.0"
 urllib3 = ">1.26.17"
-promptflow-azure = "<2.0.0" # Needed for remote tracking
 numpy = ">=1.22"
+promptflow-azure = { version = "<2.0.0", optional = true} # Needed for remote tracking
```

Comment:
Do we need this if customer will see error message that tells them to install pf-azure ?

Reply:
It is, generally an option just in case customer want to install all at once by using `pip install promptflow-evals[azure]`, it is just for convenience. This dependency will not be installed by default.

Reply:
Will error message show the command to run to install pf-azure ?

Reply:
Yes, I have added the appropriate error message.